test/functionalTest/cases/3727_avoid_triggering_notification_if_metadata_changes_but_not_the_attribute_value_itself/notifyOnMetadataChange_false_in_condition.test

```diff
@@ -0,0 +1,229 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Add also a .test for the basic notifyOnMetadataChange CRUD. For instance:

* Create entity without notifyOnMetadataChange (defaults to true)
* Modify entity with notifyOnMetadataChange: false
* Modify entity with notifyOnMetadataChange: true
* Modify entity with notifyOnMetadataChange: false
* Modify entity witout notifyOnMetadataChange (defaults to true)

Reply:
s/entity/subscription/ above :)

Reply:
Fixed in [aee4daa](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/aee4daacf51d22caf4a6b9c508f6245feb152793)